scripts/generate_landings_risk_report.py

```diff
@@ -1546,6 +1546,9 @@ def get_top_crashes(team: str, channel: str) -> str | None:
                     fix_time_diff = f"This is {verb} when compared to two weeks ago (median was {prev_median_fix_time} days)."
                 else:
                     fix_time_diff = ""
+            else:
+                median_fix_time_text = ""
+                fix_time_diff = ""
```

Comment:
nit: It could be more readable to define all the variables that have potential values to be `"` or `None` at the top; then, their values will be changed if they match any of the conditions. With that, we will have fewer need `else` statements.